lib/bike_brigade/locations/location.ex

```diff
@@ -62,7 +62,7 @@ defmodule BikeBrigade.Locations.Location do
     end
   end
 
-  defp parse_unit(address) when is_binary(address) do
+  def parse_unit(address) when is_binary(address) do
```

Comment:
We didn't end up having to export this
```suggestion
  defp parse_unit(address) when is_binary(address) do
```